enclosure.kcl

```diff
@@ -0,0 +1,159 @@
+// Enclosure
+// An enclosure body and sealing lid for storing items
+
+
+const length = 175
+const width = 125
+const height = 70
+const wallThickness = 3
+const holeDia = 4
+
+const sketch001 = startSketchOn('XY')
+  |> startProfileAt([0, 0], %)
+  |> angledLine([0, width], %, $rectangleSegmentA001)
+  |> angledLine([
+       segAng(rectangleSegmentA001) + 90,
+       length
+     ], %, $rectangleSegmentB001)
+  |> angledLine([
+       segAng(rectangleSegmentA001),
+       -segLen(rectangleSegmentA001)
+     ], %, $rectangleSegmentC001)
+  |> lineTo([profileStartX(%), profileStartY(%)], %, $rectangleSegmentD001)
+  |> close(%)
+const extrude001 = extrude(height, sketch001)
+  |> fillet({
+       radius: wallThickness * 4,
+       tags: [
+         getNextAdjacentEdge(rectangleSegmentA001),
+         getNextAdjacentEdge(rectangleSegmentB001),
+         getNextAdjacentEdge(rectangleSegmentC001),
+         getNextAdjacentEdge(rectangleSegmentD001)
+       ]
+     }, %)
+  |> shell({
+       faces: ["end"],
+       thickness: wallThickness
+     }, %)
+
+fn function001 = (originStart) => {
+  // Defining a plane to sketch on shell
+  const plane001 = {
+    plane: {
+      origin: [0.0, 0.0, wallThickness],
+      xAxis: [1.0, 0.0, 0.0],
+      yAxis: [0.0, 1.0, 0.0],
+      zAxis: [0.0, 0.0, 1.0]
+    }
+  }
+
+  // Define interior bodies to secure fasteners
+  const sketch002 = startSketchOn(plane001)
+    |> circle([originStart[0], originStart[1]], holeDia + wallThickness, %)
+    |> hole(circle([originStart[0], originStart[1]], holeDia, %), %)
+  const extrude002 = extrude(height - wallThickness, sketch002)
+
+  return extrude002
+}
+
+function001([
+  wallThickness * 3 + holeDia,
+  wallThickness * 3 + holeDia
+])
+function001([
+  wallThickness * 3 + holeDia,
+  length - (wallThickness * 3 + holeDia)
+])
+function001([
+  width - (wallThickness * 3 + holeDia),
+  wallThickness * 3 + holeDia
+])
+function001([
+  width - (wallThickness * 3 + holeDia),
+  length - (wallThickness * 3 + holeDia)
+])
+
+// Define lid position and outer surface
+const sketch003 = startSketchOn('XY')
```

Comment:
The model looks great, just add short descriptive comments like this to the other features (sketch001, function001 etc) and then it's good!